### PR TITLE
[ADD] sale_payment_method_transaction_id: link module between sale_payment_method and base_transaction_id

### PR DESCRIPTION
**Pull request by guewen**
Original proposal on: https://code.launchpad.net/~camptocamp/e-commerce-addons/7.0-sale_payment_method-base_transaction_id/+merge/203694

### CODE REVIEW

File: sale_payment_method_transaction_id/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+##############################################################################
+
+{'name' : 'Sale Payment Method - Transaction ID Compatibility',
+ 'version' : '1.0',
+ 'author' : 'Camptocamp',
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'category': 'Hidden',
+ 'depends' : ['sale_payment_method',
+              'base_transaction_id',  # in lp:banking-addons/bank-statement-reconcile-7.0
```
Comment (yvaucher) [inline sale_payment_method_transaction_id/__openerp__.py]: This should be updated showing new git repo

**Comment by yvaucher** [comment]
Otherwise :+1:

**Comment by guewen** [comment]
@yvaucher done, rebased and some pep8 fixes in the manifest

**Comment by yvaucher** [comment]
It seems the dependency is missing on travis:

except_orm: (u'Error', u"You try to install module 'sale_payment_method_transaction_id' that depends on module 'base_transaction_id'.\nBut the latter module is not available in your system.")

.travis.yml to be updated by adding bank-statement-reconcile repo

**Comment by guewen** [comment]
The build fails due to lines longer than 80 in other modules, fixed in #9

**Comment by guewen** [comment]
Rebased, green.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**